static/templates/search_link_404.html

```diff
@@ -0,0 +1,6 @@
+<!-- Printing the search link for the article that wasnt found / a full search for the article. -->
+{{#pattern}}
+<p>
+	Make a full text search for <a href="{{root}}/search?content={{{bookName}}}&pattern={{{pattern}}}">{{pattern}}</a>
```

Comment:
I think it makes sense to pass the entire search URL as a separate parameter. Then you can avoid specifying the book name when it is not needed.